manager.cpp

```diff
@@ -25,7 +25,9 @@
 #include <filesystem>
 #include <iostream>
 
-using namespace std::literals::chrono_literals;
+using namespace std;
```

Comment:
all the existing code in the file already uses the 'std::', so probably better to continue with that and just remove this line.